.github/ISSUE_TEMPLATE/offboard-team-member.md

```diff
@@ -0,0 +1,82 @@
+---
+name: "👋🏻 Offboard a departing team member"
+about: Steps to follow for a departing team member
+labels: "type: offboard"
+title: "Departing <name>"
+---
+
+- Departing person's name: `INSERT NAME`
+- Departing person's manager: `@INSERT_HANDLE`
+
+### Departure checklist
+
+The manager should follow the checklist below to offpord a departing team member after exit. Don't hesitate to delegate some actions to others (e.g. if another person needs to give access to an account because you aren't able to).
```

Comment:
```suggestion
The manager should follow the checklist below to offboard a departing team member after exit.
Don't hesitate to delegate some actions to others (e.g. if another person needs to give access to an account because you aren't able to).
```